### ISSUE TITLE
Changelog?

### ISSUE THREAD

**Issue by nazar-pc** [alexei/sprintf.js#133]
Could you add `changelog.md` or at least release notes on GitHub about meaningful changes?
I see that 1.1.0 is significantly bigger than 1.0.3, I'd like to know if it fixes anything.

In particular I don't like `Named arguments`, since I was using this library as minimalistic implementation of `sprintf`, which doesn't seem to be the case anymore.

**Comment by alexei** [comment]
I used to maintain a changelog, but I dropped it when I started using Git.

The big bump from 1.0.3 to 1.1.0 was due to the fact that I haven't released a new version since 2015 even though I and other contributors added the `t`, `T` and `v` specifiers and fixed a couple of bugs here and there. It's a minor release, so it's backwards compatible.

This library supports named arguments since at least 2011, so you were most certainly using it without knowing that feature existed. Sorry to disappoint.
